src/swipe-navigation.ts

```diff
@@ -577,56 +577,51 @@ class swipeManager {
       const tabs = this.#getTabsArray();
 
       if (view != null) {
-        if (Config.animate == "swipe") {
-          const _in = directionLeft ? `${screen.width / 1.5}px` : `-${screen.width / 1.5}px`;
-          const _out = directionLeft ? `-${screen.width / 1.5}px` : `${screen.width / 1.5}px`;
-          view.style.transitionDuration = "200ms";
-          view.style.opacity = "0";
-          view.style.transform = `translate(${_in}, 0)`;
-          view.style.transition = "transform 0.20s, opacity 0.18s";
-          setTimeout(function () {
-            tabs[index].dispatchEvent(new MouseEvent("click", { bubbles: false, cancelable: true }));
-            view.style.transitionDuration = "0ms";
-            view.style.transform = `translate(${_out}, 0)`;
-            view.style.transition = "transform 0s";
-          }, 210);
-          setTimeout(function () {
-            view.style.transitionDuration = "200ms";
-            view.style.opacity = "1";
-            view.style.transform = "translate(0px, 0)";
-            view.style.transition = "transform 0.20s, opacity 0.18s";
-          }, 250);
-        } else if (Config.animate == "fade") {
-          view.style.transitionDuration = "200ms";
-          view.style.transition = "opacity 0.20s";
-          view.style.opacity = "0";
-          setTimeout(function () {
-            tabs[index].dispatchEvent(new MouseEvent("click", { bubbles: false, cancelable: true }));
-            view.style.transitionDuration = "0ms";
+
+        if (Config.animate == "none") {
+          tabs[index].dispatchEvent(new MouseEvent("click", { bubbles: false, cancelable: true }));
```

Comment:
Ups... I think i confused "none" with undefined (i.e. no animation set in the config)
Good catch!